### ISSUE TITLE
Variables are not interpolated in xcodeproj

### ISSUE THREAD

**Issue by ianbytchek** [SiarheiFedartsou/fastlane-plugin-versioning#4]
I have a project with `INFOPLIST_FILE` containing variables inside of it and apparently Xcodeproj [doesn't](https://github.com/CocoaPods/Xcodeproj/issues/423) support interpolation, so I'm getting this:

```log
[01:34:14]: undefined method `[]' for nil:NilClass
[01:34:14]: Unable to find plist file at '…/$(inherited)/Project.plist'
```

On the other hand `FastlaneCore::Project::build_settings` uses the suggested approach in the above issue and provides correctly interpolated results. Can you please consider using that instead?

**Comment by SiarheiFedartsou** [comment]
@ianbytchek Good notice! Thanks. I'll try to migrate to `FastlaneCore::Project::build_settings`.

**Comment by SiarheiFedartsou** [comment]
Resolved in version 0.2.6. 
Now all used targets must be shared(its `FastlaneCore::Project` requirement) but I think it is not a problem because usually all targets that used with fastlane are shared.
